src/App.js

```diff
@@ -1,26 +1,87 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import { firebase, db } from './lib/firebase';
+import { useCollection } from 'react-firebase-hooks/firestore';
+
+const App = () => {
+  const [value, loading, error] = useCollection(
+    firebase.firestore().collection('list'),
```

Comment:
In `lib/firebase.js`, you have 
```
let db = firebase.firestore();
```
Since `db` is the same as `firebase.firestore()` and you imported it, you can use 
```
db.collection('list'),
```

Reply:
making this change breaks the code, unfortunately.

Reply:
<img width="913" alt="Screen Shot 2021-04-09 at 8 13 19 PM" src="https://user-images.githubusercontent.com/23660204/114277113-de457d80-99f7-11eb-9419-a993753c6e82.png">

Reply:
ugh...sorry. wrong screenshot
<img width="1055" alt="Screen Shot 2021-04-10 at 12 28 43 PM" src="https://user-images.githubusercontent.com/23660204/114277213-4b591300-99f8-11eb-9231-4898f7a1b9e2.png">